checkov/common/output/report.py

```diff
@@ -325,6 +325,7 @@ def get_sarif_json(self, tool: str) -> Dict[str, Any]:
                 "ruleId": record.check_id,
                 "ruleIndex": idx,
                 "level": level,
+                "attachments": [{'description': detail} for detail in record.details],
```

Comment:
lets go with that, even `attachments` doesn't actually fit, but there is nothing else 😄